submissions/2025-07-25-OpenFF-Lipid-Torsion-Drives-v4.0/README.md

```diff
@@ -0,0 +1,61 @@
+# OpenFF Lipid TorsionDrives v4.0
+
+### Description
```

Comment:
Thanks for updating this, it looks great. Don't forget to add the level of theory used, elements covered, charges, min/mean/max molecular weight, you'll mostly just need to copy it from the metadata section of this document.